package.json

```diff
@@ -22,6 +22,7 @@
     "@turf/turf": "^4.7.3",
     "bluebird": "^3.5.1",
     "express": "^4.16.2",
+    "pinkie": "^2.0.4",
```

Comment:
Can you remove this change? It has nothing to do with this PR.
Same goes for any changes to `package-lock.json`